admin/category_product_listing.php

```diff
@@ -17,6 +17,8 @@
 
 $search_parameter = $search_result ? '&search=' . $_GET['search'] : '';
 $keywords = $search_result ? zen_db_input(zen_db_prepare_input($_GET['search'])) : '';
+$max_results = MAX_DISPLAY_RESULTS_CATEGORIES; 
+if (!empty($search_parameter)) $max_results = MAX_DISPLAY_SEARCH_RESULTS; 
```

Comment:
i believe PSR-2 (which i think we follow) requires the use of braces for `if` control statements.  (see 5.1 on https://www.php-fig.org/psr/psr-2/).  (it is my preference as well, else i would not bring it up.)